include/dqrobotics/interfaces/coppeliasim/DQ_CoppeliaSimInterfaceZMQ.h

```diff
@@ -74,18 +74,7 @@ class DQ_CoppeliaSimInterfaceZMQ: public DQ_CoppeliaSimInterface
                                const VectorXd& torques) override;
     VectorXd get_joint_torques(const std::vector<std::string>& jointnames) override;
 
-    std::string _get_object_name(const int& handle);
-
-    template<typename T>
-    std::vector<std::string> _get_object_names(const T &handles)
```

Comment:
@juanjqo 

If the input is also `std::vector` it's better to use `std::vector<T>`

Reply:
Fixed! =)